frontend/public/components/conditions.tsx

```diff
@@ -11,15 +10,17 @@ export const Conditions: React.SFC<ConditionsProps> = ({obj}) => {
       {condition.status}
     </div>
     <div className="col-xs-3 col-sm-3">
-      {condition.reason}
+      {condition.reason ? condition.reason : '-'}
     </div>
-    <div className="col-xs-3 col-sm-5">
-      {condition.message}
+    <div className="col-xs-3 col-sm-5 co-pre-line-wrap">
+      {/* remove initial newline which appears in route messages */}
+      {condition.message ? condition.message.replace(/^.*?\r?\n/, '') : '-'}
     </div>
   </div>);
 
   return <React.Fragment>
-    <h1 className="co-section-title">Conditions</h1>
+    {title && <h1 className="co-section-title">{title}</h1>}
+    {subTitle && <h4 style={{paddingTop: 4}}>{subTitle}</h4>}
```

Comment:
Let's not use inline styles. (Should this have a unit?)

Reply:
I'd rather just remove the title from this component and make it only the table instead of having title / subTitle props